lib/src/header_parser/parser.dart

```diff
@@ -70,8 +70,9 @@ List<Binding> parseToBindings() {
   }
 
   _logger.fine('CompilerOpts used: $compilerOpts');
-  clangCmdArgs = createDynamicStringArray(compilerOpts);
-  final cmdLen = compilerOpts.length;
+  clangCmdArgs = createDynamicStringArray(
+      [...compilerOpts, '-I', '/usr/lib/clang/13.0.1/include/']);
```

Comment:
Can this path be found automatically?

For example with https://github.com/dart-lang/ffigen/blob/master/lib/src/config_provider/path_finder.dart

Also, I believe the headers that we use are in https://github.com/dart-lang/ffigen/tree/master/third_party/libclang/include/clang-c

Reply:
Oh, you already beat me to it! 😄